data_steward/cdr_cleaner/cleaning_rules/deid/remove_fitbit_data_if_max_age_exceeded.py

```diff
@@ -0,0 +1,171 @@
+"""
+Remove all FitBit data for participants exceeding the maximum age of 89
+
+Original Issue: DC-1001
+
+The intent is to ensure there is no data for participants over the age of 89 in
+Activity Summary, Heart Rate Minute Level, Heart Rate Summary, and Steps Intraday tables
+by sandboxing the applicable records and then dropping them.
+"""
+
+# Python Imports
+import logging
+
+# Third Party Imports
+from jinja2 import Environment
+
+# Project Imports
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.bq_utils import WRITE_TRUNCATE
+
+LOGGER = logging.getLogger(__name__)
+
+jinja_env = Environment(
+    # help protect against cross-site scripting vulnerabilities
+    autoescape=True,
+    # block tags on their own lines
+    # will not cause extra white space
+    trim_blocks=True,
+    lstrip_blocks=True,
+    # syntax highlighting should be better
+    # with these comment delimiters
+    comment_start_string='--',
+    comment_end_string=' --')
+
+TABLES = [
+    'activity_summary', 'heart_rate_minute_level', 'heart_rate_summary',
+    'steps_intraday'
+]
+
+# Save rows that will be dropped to a sandboxed dataset
+SAVE_ROWS_TO_BE_DROPPED_QUERY = jinja_env.from_string("""
+CREATE OR REPLACE TABLE `{{project}}.{{sandbox_dataset}}.{{sandbox_table}}` AS
+SELECT * FROM `{{project}}.{{dataset}}.{{table}}`
+WHERE person_id IN (
+  SELECT person_id
+    FROM (
+    SELECT DISTINCT person_id,
+        EXTRACT(YEAR FROM CURRENT_DATE()) - year_of_birth AS age
+            FROM `{{project}}.{{dataset}}.person` ORDER BY 2)
+        WHERE age > 89)
```

Comment:
@dcampbell-vumc This is not going to be the exact same list as what is used by deid.  So, there will potentially be a small subset of people in this table that do not make it into deid.  


You are sandboxing anyone older than 89.  `curation/data_steward/tools/run_deid.py` is using `age < {{max_age}}` where max_age is a parameter.  So, your sandbox won't include those people who are 89 and they will be included into the released fitbit data.  deid will not include these people in deid tables.  So they don't need to be present here.

Reply:
Should I parameterize max age? I went ahead and changed it to `WHERE age >= 89` and tested it to make sure that those who are exactly 89 are sandboxed and dropped, so it won't be too much more work to parameterize max age.

Reply:
Thanks @lrwb-aou for finding this. I didn't cross-check this with current deid.